src/easyscore.ts

```diff
@@ -227,7 +228,7 @@ export class Piece {
 }
 
 export interface BuilderElements {
-  notes: StaveNote[];
+  notes: StemmableNote[];
```

Comment:
Not sure I understand why we're restricting to `StemmableNote` -- what do we gain from this?

Reply:
Support for GhostNotes in EasyScore. `GhostNote` is a `StemmableNote`.
Please note that this restriction appeared in `EasyScore`, a `Voice` accepts `Tickable`